nextjs_frontend/src/app/api/auth/[...nextauth]/options.ts

```diff
@@ -0,0 +1,40 @@
+import type { NextAuthOptions } from "next-auth";
+import GoogleProvider from "next-auth/providers/google";
+import CredentialsProvider from "next-auth/providers/credentials";
+
+export const options: NextAuthOptions = {
+  providers: [
+    GoogleProvider({
+      clientId: process.env.GOOGLE_ID as string,
+      clientSecret: process.env.GOOGLE_SECRET as string,
```

Comment:
These variables aren't specified on README. You need to add to env.example and explain that it's neeeded these variables to work

Reply:
ok!